user_modules/shared_testcases/commonFunctions.lua

```diff
@@ -1230,4 +1231,59 @@ function commonFunctions:update_json_file(path_to_json, old_section, new_section
   file:close()
 end
 
+-- ---------------------------------------------------------------------------------------------
+--27. Function joins paths of file system
+-- ---------------------------------------------------------------------------------------------
+--! @brief Return the path resulting from combining the individual paths or nil
+--! if the second (or later) path is absolute will be returned the last absolute path (joined with any non-absolute paths following).
+--! empty elements (except the first) will be ignored.
+--! @string p1 A file path
+--! @string p2 A file path
+--! @string ... more file paths
+function commonFunctions:path_join(p1, p2, ...)
```

Comment:
@VVeremjova, Usage example was added to description (see 4e658b7e6cd3feb990de6a606e693c8760e94170)